lib/ansible/modules/network/nxos/nxos_interface_ospf.py

```diff
@@ -53,6 +53,13 @@
         Valid values are a string, formatted as an IP address
         (i.e. "0.0.0.0") or as an integer.
     required: true
+  bfd:
+    description:
+      - Enables bfd at interface level. This overrides the bfd variable set at the ospf router level.
+      - Valid values are 'true', 'false' (disable bfd on this interface), or 'default'.
+      - "Dependency: 'feature bfd'"
+    version_added: "2.9"
+    type: str
```

Comment:
Since type is `str` how about `enable`, `disable`, `default`. Because there are PRs where bfd type is bool which accepts true and false, and `str` also accepts bool value.
I believe `enable/disable`, bfd can be consistent across modules.

Reply:
Sounds good. I'll change it here and on the other interface-related modules that have this property.

Reply:
Thanks!